examples/demo-app/src/cloud-providers/aws/aws-login.js

```diff
@@ -0,0 +1,25 @@
+import React, {useEffect} from 'react';
+import Amplify, {Hub} from 'aws-amplify';
+import awsconfig from './aws-exports';
```

Comment:
shouldn't it be awsmobile? why is this working?

Reply:
No idea, it was automatically imported like this and working.